build-logic-commons/src/main/kotlin/gradlebuild.build-logic.groovy-dsl-gradle-plugin.gradle.kts

```diff
@@ -28,7 +28,7 @@ java {
 
 dependencies {
     api(platform(project(":build-platform")))
-    implementation("gradlebuild:code-quality")
+    implementation("gradlebuild:build-logic-commons")
```

Comment:
There still should be at least one subproject in `build-logic-commons`, probably called `build-logic-commons`. There shouldn't be a `src` directory in the root ` build-logic-commons`. The reason there is that if we need to add more subprojects for some reason later, then it would be easy to do so.